src/App.vue

```diff
@@ -1,26 +1,11 @@
+<script setup>
+import AppHeader from '@/components/atoms/AppHeader.vue'
+import MainContent from '@/components/templates/MainContent.vue'
+</script>
+
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
-  <HelloWorld msg="Welcome to Your Vue.js App"/>
+  <AppHeader />
```

Comment:
I think this can be moved to a new Page (main page) and used in App.vue. This would be more consistent with atomic design.